### DISCUSSION TITLE
Have the history changes more visible

### DISCUSSION THREAD

**Discussion by Limezy** [outline/outline#3198] [Ideas]
Hi !

As I'm working with several people on the same doc, but not synchronously, some visible tracking of history with additions and deletions would be very nice. Just like what s shown in the below screenshot (not mine - extracted from the link below).

![1*tSuAfd7GowO1oQoLRPQt5A](https://user-images.githubusercontent.com/24638389/156554313-52c953f8-4538-4d06-bdad-4d03e40e5ea3.gif)

https://open.nytimes.com/building-a-text-editor-for-a-digital-first-newsroom-f1cb8367fc21

**Comment by ReessKennedy** [comment]
+1 ... I support this too. I think better change tracking would be great on the page level and global changes that global admins can track. I haven't tried the Slack integration and I realize this will push updates or changes but for those who don't use slack it'd be nice to have more change tracking built into core.

**Comment by daladim** [comment]
I think this feature has been added into 0.67, and this discussion could be resolved.
